docs/releases.md

```diff
@@ -30,6 +30,17 @@
 
 ## Master branch
 
+Lychee currently undergoes a major architectural refactoring of its internals.
+The master branch already includes the first part of that refactoring which mostly concentrates on the backend.
+
+These changes and information only affect those users who directly follow the master branch:
+
+- Despite the substantial changes your photos and albums should be as safe as with any other commit to the master branch. Don't worry.
+- As we did our best to ensure that Lychee behaves as before, you might potentially encounter more regression bugs than usually. These bugs are _not_ expected to severely affect the overall functioning of Lychee, but may be annoying.
+- **New ID scheme:** Albums and photos are migrated to a new ID scheme. Previously, albums and photos used time-based integer IDs. The new IDs are truly 144bit randomness encoded in Base64 as a 24-character string, e.g., if your photo had the URL [https://lycheeorg.github.io/r/16102925744307/16102927818284](https://lycheeorg.github.io/demo/#16102925744307/16102927818284) before, it may have the ID [https://lycheeorg.github.io/r/GTqZfSso3nPeCnTNW4ovisgC/jM6KkmlK7X0LtVas5MjrHtTO](https://lycheeorg.github.io/demo/#16102925744307/16102927818284) after the migration. This means, **external links to your albums or photos will stop working.** However, we implemented a redirection service which _is disabled by default_. You can enable the re-direction service under `Settings` > `More` > `legacy_id_redirection`. This gives you time to migrate any external reference to the new IDs. We also generate a log entry every time the redirection service is used. This log contains the legacy and new ID as well as information where the request came from. This may help you with the migration of your external links.
```

Comment:
144bit _of_ randomness (?)

Reply:
"The new IDs are 24-character base64-encoded strings." ?

Is it worth the URLs being tagged, especially if one of them is going to break immediately?

Also, I still lean towards enabling the redirect by default.